Sources/From.swift

```diff
@@ -149,7 +149,8 @@ public struct From<O: DynamicObject> {
             )
             throw CoreStoreError.unknown
         }
-        fetchRequest.entity = parentStack.entityDescription(for: Internals.EntityIdentifier(self.entityClass))!
+        guard let entity = parentStack.entityDescription(for: Internals.EntityIdentifier(self.entityClass)) else { return }
+        fetchRequest.entity = entity
```

Comment:
What specific use-case does this resolve?

Reply:
I was facing an error when debugging, dunno if related to change of the store in a secure way, so I preferred to guard and avoid a strong crash